tests/core/process.c

```diff
@@ -79,6 +79,13 @@ struct _TestRangeContext
   gboolean found_exact;
 };
 
+struct _TestThreadSyncData
+{
+  GMutex thread_is_started_mutex;
+  GCond thread_is_started;
```

Comment:
Let's rename these to `mutex` and `cond`, and introduce `gboolean started;` or something like that.